src/uwtools/tests/fixtures/hello_workflow_tags.yaml

```diff
@@ -0,0 +1,11 @@
+  workflow:
```

Comment:
Compared to `hello_workflow` which contains task, metatask and dependency tags, `hello_workflow_tags` has to include `workflow` to pass (as of right now). This is because within the `rocoto.jsonschema` we include the `workflow` tag.